### PR TITLE
DOC: Fix typos in the random and f2py documentation.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by melissawm** [comment]
The typo fixes look good, can you remove the .replit file this added? Thanks

**Comment by MalikIdreesHasanKhan** [comment]
OK, sure.

File: numpy/ma/tests/test_extras.py [hunk 1]
```diff
@@ -1580,9 +1580,9 @@ def test_atleast_2d(self):
         assert_equal(b.mask.shape, b.data.shape)
 
     def test_shape_scalar(self):
-        # the atleast and diagflat function should work with scalars
+        # the at least and diagflat function should work with scalars
```
Comment (charris) [inline numpy/ma/tests/test_extras.py:1583]: `atleast` is a function name, so it should not be corrected. More precisely, it turns up as `atleast_(1d|2d|3d)`

Comment (MalikIdreesHasanKhan) [reply numpy/ma/tests/test_extras.py]: Did as you said, waiting for review :).

**Comment by bashtage** [comment]
Ideally you should rebase and squash so that this unnecessary file doesn't enter the repo as an object.

**Comment by mattip** [comment]
Thanks @MalikIdreesHasanKhan

**Comment by MalikIdreesHasanKhan** [comment]
My pleasure @mattip

**Pull request merged**